code/go/0chain.net/chaincore/chain/handler_integration_tests.go

```diff
@@ -34,18 +35,18 @@ func LatestFinalizedBlockHandler(ctx context.Context, r *http.Request) (
 }
 
 /*LatestFinalizedMagicBlockHandler - provide the latest finalized magic block by this miner */
-func LatestFinalizedMagicBlockHandler(ctx context.Context, r *http.Request) (
-	interface{}, error) {
-
-	var state = crpc.Client().State()
-	if state.MagicBlock != nil {
```

Comment:
Think, it can possibly break the `"magic_block"` directive's logic.
I've tried to fix it and run `conductor.sharders.yaml` tests, but it seems like it was broken before.